diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/kdoc/KdocComments.kt

```diff
@@ -45,7 +62,148 @@ class KdocComments(private val configRules: List<RulesConfig>) : Rule("kdoc-comm
             when (node.elementType) {
                 FILE -> checkTopLevelDoc(node)
                 CLASS -> checkClassElements(node)
+                VALUE_PARAMETER -> checkValueParameter(node)
+            }
+    }
+
+    @Suppress("UnsafeCallOnNullableType")
+    private fun checkValueParameter(node: ASTNode) {
+        if (node.parents().none { it.elementType == PRIMARY_CONSTRUCTOR } ||
+                !(node.hasChildOfType(VAL_KEYWORD) || node.hasChildOfType(VAR_KEYWORD))) return
+        val prevComment = if (node.treePrev.elementType == WHITE_SPACE &&
+                (node.treePrev.treePrev.elementType == EOL_COMMENT ||
+                        node.treePrev.treePrev.elementType == BLOCK_COMMENT)) {
+            node.treePrev.treePrev
+        } else if (node.hasChildOfType(KDOC)) {
+            node.findChildByType(KDOC)!!
+        } else if (node.treePrev.elementType == BLOCK_COMMENT) {
+            node.treePrev
+        } else {
+            null
+        }
+        val kDocBeforeClass = node.parent({ it.elementType == CLASS })!!.findChildByType(KDOC)
+        if (prevComment != null) {
+            if (kDocBeforeClass != null)
+                checkKDocBeforeClass(node, kDocBeforeClass, prevComment)
+            else
+                createKDocWithProperty(node, prevComment)
+        } else {
+            if (kDocBeforeClass != null)
+                checkBasicKDocBeforeClass(node, kDocBeforeClass)
+            else
+                createKDocBasicKDoc(node)
+        }
+    }
+
+    @Suppress("UnsafeCallOnNullableType")
+    private fun checkBasicKDocBeforeClass(node: ASTNode, kDocBeforeClass: ASTNode) {
+        val propertyInClassKDoc = kDocBeforeClass
+                .kDocTags()
+                ?.firstOrNull { it.knownTag == KDocKnownTag.PROPERTY && it.getSubjectName() == node.findChildByType(IDENTIFIER)!!.text }
+        if (propertyInClassKDoc == null && node.getFirstChildWithType(MODIFIER_LIST).isAccessibleOutside()) {
+            KDOC_NO_CONSTRUCTOR_PROPERTY.warnAndFix(configRules, emitWarn, isFixMode,
+                    "add ${node.findChildByType(IDENTIFIER)!!.text} in KDoc", node.startOffset, node) {
```

Comment:
```suggestion
                    "add <${node.findChildByType(IDENTIFIER)!!.text}> to KDoc", node.startOffset, node) {
```
it'll be more readable if property name is marked somehow, for example with `<...>`